ruma-federation-api/src/keys/claim_keys/v1.rs

```diff
@@ -0,0 +1,38 @@
+//! [POST /_matrix/federation/v1/user/keys/claim](https://matrix.org/docs/spec/server_server/r0.1.4#id91)
+
+use std::collections::BTreeMap;
+
+use ruma_api::ruma_api;
+use ruma_identifiers::{DeviceId, DeviceKeyAlgorithm, DeviceKeyId, UserId};
+use serde::{Deserialize, Serialize};
+
+ruma_api! {
+    metadata: {
+        description: "Claims one-time keys for use in pre-key messages.",
+        method: POST,
+        name: "claim_keys",
+        path: "/_matrix/federation/v1/user/keys/claim",
+        rate_limited: false,
+        requires_authentication: false,
+    }
+
+    request: {
+        /// The keys to be claimed.
+        /// #[ruma_api(path)]
```

Comment:
Seems like a copy-paste error.

Reply:
I also realized this should be `#[ruma_api(body)]` instead

Reply:
No, it shouldn't actually. `#[ruma_api(body)]` would be for when the request body would be `{ "<user id 1>": { … }, "<user id 2>": { … } }`, but it's `{ "one_time_keys": { "<user id 1>": { … }, "<user id 2>": { … } } }`

Reply:
Should it just not have an attribute in that case?

Reply:
Yeah. Regular body fields don't have attributes. (see also [ruma_api! docs](https://docs.rs/ruma-api/0.17.0-alpha.1/ruma_api/macro.ruma_api.html)